src/thunderbots/software/ai/hl/stp/action/kick_action.h

```diff
@@ -37,6 +37,7 @@ class KickAction : public Action
      * Returns the next Intent this KickAction wants to run, given the parameters.
      *
      * @param robot The robot that should perform the kick
+     * @param ball The ball being kicked
      * @param kick_origin The location where the kick will be taken
```

Comment:
See above comment about doc'ing what `ball` and `kick_origin` are used for respectively.

Reply:
fixed